frontend/src/home/Home.tsx

```diff
@@ -492,6 +492,40 @@ class HomeComponent extends React.Component<Props, HomeState> {
     }
   }
 
+  /**
+   * If a user is currently logged in, saves the current plan under this user.
+   * Only supports updating a user's singular plan, can be modified later to
+   * update a specific plan.
+   */
+  savePlan() {
+    // If this is true, then a user is currently logged in and we can update their plan
+    if (this.props.token && this.props.userId) {
```

Comment:
You can use` this.props.isLoggedIn` here to check if user is logged in, just so its more readable :)